conf/app.routes

```diff
@@ -17,7 +17,12 @@ POST        /who-is-disclosing               controllers.UserTypeController.onSu
 GET         /one-or-more-entries             controllers.NumberOfEntriesController.onLoad
 POST        /one-or-more-entries             controllers.NumberOfEntriesController.onSubmit
 
+GET         /unauthorised          controllers.errors.UnauthorisedController.onPageLoad
```

Comment:
Can you sort out formatting in this file please?

Reply:
done